src/librustdoc/src/lib.rs

```diff
@@ -741,30 +740,35 @@ where R: 'static + Send,
             }
         }
 
-        // Load all plugins/passes into a PluginManager
-        let path = plugin_path.unwrap_or("/tmp/rustdoc/plugins".to_string());
-        let mut pm = plugins::PluginManager::new(PathBuf::from(path));
+        if !plugins.is_empty() {
+            eprintln!("WARNING: --plugins no longer functions; see CVE-2018-1000622");
+        }
+
+        if !plugin_path.is_none() {
```

Comment:
Since this is printing the sale as the previous `if` condition, could you merge the both please?

Reply:
They don't; the name of the command is in the warning.

Reply:
Good point, my bad!